README.md

```diff
@@ -1,128 +1,276 @@
+*Note: This is Neo 3 branch, currently under development. For the current stable version, please [click here]()*
 
-NEO 2.x: Reference template  
-================
+## Features
+These are a few features Neo has:
 
-NEO adopts a model of continuous improvement and evolution.
-We believe that the research and development of blockchain technology will be kept in continuous transformation for the next years.
-From 2017 until the middle of 2020 (estimated), NEO 2.x will still be supported and the compatibility of new packages will be maintained.
+- **[dBFT2.0](https://medium.com/neo-smart-economy/neos-dbft-2-0-single-block-finality-with-improved-availability-6a4aca7bd1c4)**
+  - Single block finality consensus algorithm.
+- **[Smart Contracts using C#](https://github.com/neo-ngd/NEO3-Development-Guide/tree/master/en/SmartContract)**
+  - Build smart-contracts using C# sintax;
+  - Python, Typescript and Go Smart Contracts provided by community projects.
+- **[Unity support]()**
+  - Neo can be used to create your game in the blockchain.
+- **[Neo Blockchain Toolkit for .NET]()**
+  - Developer tools, supporting easy smart-contract development with debugging support using Visual Studio Code.
+- **[Plugin system]()**
+  - Used to extend Neo functinalities, the plugin system allow developers to easily add new features to their nodes.
+- **[Native contracts](https://medium.com/neo-smart-economy/native-contracts-in-neo-3-0-e786100abf6e)**
+  - Contracts running C# code.
+- **[Smart Contract internet access](https://medium.com/neo-smart-economy/on-the-importance-of-oracles-neo-3-0-and-dbft-17c37ee35f32)**
+  - Internet acess during a transaction.
+- **[Voting Mechanism](https://medium.com/neo-smart-economy/how-to-become-a-consensus-node-27e5317722e6)**
+  - Decentralizing control over the network by allowing NEO holders to vote for consensus nodes.
+- **[Distributed file-system]()**
+  - NeoFS is a scalable, decentralized object storage network integrated with NEO contracts to provide trustless data storage facilities.
+- **[Digital identity]()**
+  - Using trust, privacy and game theory models. (WIP)
 
-A link to the essential C# reference implementation can be seen below:
 
-* [neo](https://github.com/neo-project/neo/tree/master-2.x): core implementation (branch `master-2.x`)
-* [neo-vm](https://github.com/neo-project/neo-vm/tree/master-2.x): virtual machine (branch `master-2.x`)
-* [neo-cli](https://github.com/neo-project/neo-cli/tree/master-2.x): command-line interface (branch `master-2.x`)
-* [neo-plugins](https://github.com/neo-project/neo-plugins/tree/master-2.x): plugins (branch `master-2.x`)
-* [neo-devpack-dotnet](https://github.com/neo-project/neo-devpack-dotnet/tree/master-2.x): NeoContract compiler and development pack for .NET (branch `master-2.x`)
+## Quick Start
 
-NEO 1.x was known as Antshares and the roots of the code can be found historically in this current repository.
+#### Building a smart-contract
```

Comment:
this part is mainly for devpack and neon compiler, the instructions is already included in Docs. 

Also, this is more appropriate for Readme in devpack or neon compiler, right ?